corehq/apps/cloudcare/static/cloudcare/js/form_entry/fullform-ui.js

```diff
@@ -542,6 +592,29 @@ hqDefine("cloudcare/js/form_entry/fullform-ui", function () {
             if (!resourceType || !_.isFunction(Utils.resourceMap)) { return ''; }
             return Utils.resourceMap(resourceType);
         };
+
+        self.requiredNotAnswered = ko.computed(function () {
+            return self.required() && self.answer() === Const.NO_ANSWER;
+        });
+
+        self.navigateTo = function () {
+            // toggle nested collapsible Groups
+            var hasParent = self.parent !== undefined;
+            var currentNode = self;
+            while (hasParent) {
+                hasParent = currentNode.parent !== undefined;
+                var parent = currentNode.parent;
+                if (parent !== undefined && parent.collapsible !== undefined && !parent.showChildren()) {
+                    parent.toggleChildren();
+                }
+                currentNode = parent;
+            }
+            var el = $("label[for='" + self.entry.entryId + "']");
+            $('html, body').animate({
+                scrollTop: $(el).offset().top - 60,
+            });
+            el.fadeOut(200).fadeIn(200).fadeOut(200).fadeIn(200);
```

Comment:
What does this do? Make the label for the errored question fade in & out to catch the user's attention?

Reply:
It blinks the question, useful when there is only one required question and clicking the button repeatedly gives this feedback without which the button feels like doing nothing

Reply:
cool!